languages/de-DE/Misc.multids

```diff
@@ -63,6 +66,8 @@ MissingTiddler/Hint: Fehlender Tiddler "<$text text=<<currentTiddler>>/>" - klic
 No: Nein
 OfficialPluginLibrary: Offizielles ~TiddlyWiki Plugin-Verzeichnis
 OfficialPluginLibrary/Hint: Offizielles ~TiddlyWiki Plugin-Verzeichnis auf tiddlywiki.com. Plugin, Themes und Sprach Dateien werden vom "core team" gewartet.
+PageTemplate/Description: das Standard ~TiddlyWiki "Layout"
+PageTemplate/Name: Standard "~PageTemplate"
```

Comment:
I think you can drop the quotes here too, Theme and Plugin are used without quotes in the line right above, and it would be inconsistent to leave PageTemplate as the only term in quotes. Maybe translate it, I don't know yet what the TiddlyWiki conventions are about keeping the terms or translating them (e.g."Standard Seitenvorlage" would work IMO).

Reply:
PageTemplate, EditTemplate und ViewTemplate can't be translated. IMO it important to keep them. So users know what they have to search in advanced search "system" or "shadow" tabs. ..